Server/src/services/tools/manage_sprite.py

```
@@ -0,0 +1,161 @@
+            return {"success": False,
+                    "message": f"Clip name '{name}' cannot contain a path separator; "
+                               "use 'output_dir' to choose where clips are written."}
+
+    if action_lower == "setup_controller" and not controller_path:
+        return {"success": False, "message": "'controller_path' is required for setup_controller (e.g. 'Assets/Animators/Hero.controller')."}
+
+    unity_instance = await get_unity_instance_from_context(ctx)
+
+    params: dict[str, Any] = {"action": action_lower}
+
+    if path is not None:            params["path"]           = path
```

Ruff (0.16.1)

[error] Multiple statements on one line (colon)

(E701)